pom.xml

```diff
@@ -22,7 +22,7 @@
     </licenses>
 
     <properties>
-        <workflow.version>1.1</workflow.version>
+        <workflow.version>1.4.2</workflow.version>
```

Comment:
`build` did not return an appropriate value in the ancient 1.1 release. 1.4.2 is the newest compatible with 1.580.x.